Dockerfile

```diff
@@ -1,33 +1,34 @@
 FROM tomcat:9-jre8
-MAINTAINER Opengrok Community "opengrok-users@yahoogroups.com"
-ADD https://github.com/OpenGrok/OpenGrok/releases/download/1.0/opengrok-1.0.tar.gz /opengrok-1.0.tar.gz
-RUN tar -zxvf /opengrok-1.0.tar.gz
-RUN mv opengrok-* /opengrok
+MAINTAINER Nathan Guimaraes "dev.nathan.guimaraes@gmail.com"
+
+#PREPARING OPENGROK BINARIES AND FOLDERS
+ADD https://github.com/OpenGrok/OpenGrok/releases/download/1.0/opengrok-1.0.tar.gz /opengrok.tar.gz
+RUN tar -zxvf /opengrok.tar.gz && mv opengrok-* /opengrok && chmod -R +x /opengrok/bin
 RUN mkdir /src
 RUN mkdir /data
 RUN ln -s /data /var/opengrok
 RUN ln -s /src /var/opengrok/src
 
-RUN apt-get update && apt-get install -y exuberant-ctags git subversion mercurial wget inotify-tools unzip openssh-server cron
-RUN mkdir /var/run/sshd
+#INSTALLING DEPENDENCIES
+RUN apt-get update && apt-get install -y exuberant-ctags git subversion mercurial unzip openssh-server cron inotify-tools
```

Comment:
would it be possible to install universal ctags instead ?

Reply:
I'll do it and let you know when its ready.